javalib/src/main/scala/java/lang/UnixProcess.scala

```diff
@@ -26,7 +33,7 @@ private[lang] class UnixProcess private (
     outfds: Ptr[CInt],
     errfds: Ptr[CInt]
 ) extends Process {
-  override def destroy(): Unit = kill(pid, 9)
+  override def destroy(): Unit = signal.kill(pid, 9)
```

Comment:
Another pick review mode item:

Granted that anybody reading this code should understand that signal 9
means SIGKILL, using Posix signal.SIGKILL (included already) would
remove a magic number.

Same offer, if you agree with concept but want me to do the work in a further
PR, call out.

Reply:
No I will fix that.